nautobot/core/management/commands/generate_test_data.py

```diff
@@ -348,6 +357,16 @@ def _create_batch(some_factory, count, description="", **kwargs):
             has_contact=False,
             description="with teams",
         )
+        _create_batch(
+            AccessPointGroupFactory,
+            20,
+        )
+        _create_batch(SupportedDataRateFactory, 20)
+        _create_batch(RadioProfileFactory, 20)
+        _create_batch(WirelessNetworkFactory, 20)
+        _create_batch(AccessPointGroupDeviceAssignmentFactory, 10)
+        _create_batch(AccessPointGroupRadioProfileAssignmentFactory, 10)
+        _create_batch(AccessPointGroupWirelessNetworkAssignmentFactory, 10)
```

Comment:
These probably need to be moved to just after the controller factories so that we don't break the Tenant filter tests (L317)